pom.xml

```diff
@@ -48,16 +56,33 @@
             <groupId>org.projectlombok</groupId>
             <artifactId>lombok</artifactId>
             <version>${version.lombok}</version>
-            <scope>provided</scope>
         </dependency>
 
+        <dependency>
+            <groupId>joda-time</groupId>
```

Comment:
Do you think we still need it? I thought new Java Time API is very good replacement for joda time that removes need to have external library for time operations.